build/mergeallmodules.sh

```diff
@@ -0,0 +1,170 @@
+#!/bin/bash
+
+#Assumes that ./x64 and ./x86 exist and contain the arch-specific copies of a DSC resource module
+
+zip -L >/dev/null 2>&1
+if [ $? != 0 ]; then
+        echo "zip must be installed"
+        exit 1
```

Comment:
Inconsistent indentation. What are you using to edit?

Space indent for clarity, with stops every 4 bytes. This should only be indented 4 bytes.